fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/query/plan/temp/expressions/SelectExpression.java

```diff
@@ -207,15 +207,39 @@ public int hashCodeWithoutChildren() {
             return ImmutableList.of();
         }
         final Map<CorrelationIdentifier, ComparisonRange> mergedParameterBindingMap = mergedParameterBindingMapOptional.get();
+        
+        final ImmutableSet.Builder<CorrelationIdentifier> matchedCorrelatedToBuilder = ImmutableSet.builder();
+        // Loop through all child matches and reject a match if the children matches were unable to match all
+        // for each quantifiers. Also keep track of all aliases the matched quantifiers are correlated to.
```

Comment:
I think there is either a word missing here or perhaps two extra words.

Reply:
I think the difficulty in parsing this comes from the fact that I use for each quantifier as a term. Maybe for-each quantifier?